### PR TITLE
Upgrade pluto database to 19v2 to match WoW [ch6812]

### PR DESCRIPTION
**Pull request by samaratrilling**
Fixes https://app.clubhouse.io/justfixnyc/story/6626/investigate-why-ddo-explodes-on-1425-saint-nicholas-avenue. [ch6812]
[ch6626]

### Previously:
Screen Shot 2021-04-12 at 4 07 17 PM

### Now:

Screen Shot 2021-04-12 at 4 13 19 PM

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -4094,7 +4094,7 @@
               "name": "incomeAmountMonthly",
               "type": {
                 "kind": "SCALAR",
-                "name": "Float",
+                "name": "Decimal",
```
Comment (toolness) [inline schema.json:4097]: Oh shoot, I think this is the `schema.json` from #2034...  it's an auto-generated file that we keep version controlled for various reasons.  I think the change from `Float` to `Decimal` is actually due to our upgrade of the `graphene-django` package, so it's possible you're using the new version of that package on this branch... so you might want to run `bash ./docker-update.sh` to revert to the old version of that package, and then run `python manage.py graphql_schema` to regenerate the `schema.json` so it looks like the version that's currently on `master`... if that makes sense.

Err alternatively though we could just merge #2034 and then these changes will be in-sync with `master`, hehe.

Comment (samaratrilling) [reply schema.json:4097]: ahaha, you're right - i was testing your django changes and forgot i had to revert back. will revert

**Comment by samaratrilling** [comment]
I'm going to fix the exploding bug in 2 PRs: this one fixes the first address on St Nicholas and the next one will address the other address at 131 North 5 St, since they have different causes. Feel free to review this small one now.

Did some experimenting with rebasing vs merging to get this branch in line with master - seems like we don't often check out each other's branches so rebasing is not out of the question, but merging master into our feature branches and then squash merging back into master seems like it is a bit safer. Happy to hear thoughts!

**Comment by toolness** [comment]
Awesome thanks, sounds good!

Regarding merging master into our feature branches and then squash merging back into master--yup, that is what I do.  I don't usually rebase once a conversation has started, because in my experience that has required force-pushing to the branch, which can sometimes lose conversation threads in the PR--but things may have changed on GitHub since I last tried doing that, or maybe I was doing something wrong before, so happy to hear any other ideas.

**Review by toolness — APPROVED** [review]
Awesome, great detective work @samaratrilling!  Splitting it up into two separate PRs is a good idea too.

**Pull request merged**